packages/wdio-browserstack-service/src/accessibility-handler.ts

```diff
@@ -287,16 +330,18 @@ class _AccessibilityHandler {
                 )
         ) {
             BStackLogger.debug(`Performing scan for ${command.class} ${command.name}`)
-            await performA11yScan(this._browser, true, true, command.name)
+            await performA11yScan(this.isAppAutomate, this._browser, true, true, command.name)
         }
         return origFunction(...args)
     }
 
     private async sendTestStopEvent(browser: WebdriverIO.Browser, dataForExtension: any) {
         BStackLogger.debug('Performing scan before saving results')
-        await performA11yScan(browser, true, true)
-        const results: unknown = await (browser as WebdriverIO.Browser).executeAsync(accessibilityScripts.saveTestResults as string, dataForExtension)
-        BStackLogger.debug(util.format(results as string))
+        await performA11yScan(this.isAppAutomate, browser, true, true)
+        if (!isAppAccessibilityAutomationSession(this._accessibility, this.isAppAutomate)) {
```

Comment:
if (isAppAccessibilityAutomationSession) return;

Reply:
facing any issues in that?

Reply:
Done